cime_config/config_grids.xml

```diff
@@ -3265,6 +3307,14 @@
       <map name="OCN2ATM_SMAPNAME">cpl/gridmaps/ECwISC30to60E3r2/map_ECwISC30to60E3r2_to_ne30pg2_traave.20231018.nc</map>
     </gridmap>
 
+    <gridmap atm_grid="ne30np4.pg2" ocn_grid="RRSwISC6to18E3r1">
+      <map name="ATM2OCN_FMAPNAME">cpl/gridmaps/ne30pg2/map_ne30pg2_to_RRSwISC6to18E3r1_mono.230914.nc</map>
```

Comment:
This should be renamed to "traave" instead of "mono".

Reply:
@rljacob -- I'll regenerate the mapping files to match the new algorithms and naming convention